templates/include/https_params.conf

```
@@ -0,0 +1,2 @@
+{% from (role_path + '/../nginx/templates/macros/https_params.conf')|relpath(playbook_dir) import https_params with context %}
```

`roles/nginx/../nginx/`?

```suggestion
{% from (role_path + '/templates/macros/https_params.conf')|relpath(playbook_dir) import https_params with context %}
```